src/views/Feed/FeedExtra.js

```diff
@@ -14,7 +15,17 @@ function FeedExtra(props) {
   )
   const rest = getUnhandledProps(FeedExtra, props)
 
-  return <div {...rest} className={classes}>{children || text}</div>
+  if (!images || _.isBoolean(images)) {
+    return <div {...rest} className={classes}>{children || text}</div>
+  }
+
+  const imagesJSX = images.map((image, index) => {
+    const key = `${index}`
```

Comment:
In docs src's are same (`/images/wireframe/image.png`), so I have:
![image](https://cloud.githubusercontent.com/assets/14183168/17489327/2435ce3c-5da6-11e6-9bf4-05b2d89f5b46.png)

---

How I can deal with this, when I have element at input?

```jsx
const images = [<img src='http://semantic-ui.com/images/wireframe/image.png' />, ...];
```

```jsx
const imagesJSX = images.map(image => {
  return _.isString(image) ? <img key={image} src={image} /> : image
})
```

I think that `React.cloneElement(image, {key: image.src})` is smelly solution.